src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,8 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Email;
+import mate.academy.spring.validation.Password;
+
+@Password
 public class UserRequestDto {
+    @Email
     private String email;
     private String password;
```

Comment:
```suggestion
    private String email;
    @Password
    private String password;
```

Reply:
but in the checklist:
![screen1](https://github.com/mate-academy/jv-spring-security/assets/106387200/16a90d5e-02b1-4d12-925a-55d5be52add6)